apk-requirements.txt

```
@@ -0,0 +1,2 @@
+libc6-compat
+xz-dev
```

We are also missing libssl. It is installed by default in Alpine Linux, but I think is good to be explicit (specially if they change to libssl3 any time soon).
```suggestion
xz-dev
libssl1.1
```

---

👍